### PR TITLE
0x01. Caching | Initial Product

### PR DESCRIPTION
**Pull request by the1Riddle**
***Expect output errors*** on the last task 100...

still be under progress to be updated soon.

### CODE REVIEW

File: 0x01-caching/100-lfu_cache.py
Comment (the1Riddle) [inline 0x01-caching/100-lfu_cache.py:1]: expecting an output error on this file.

**Review by the1Riddle — COMMENTED** [review]
0x01. Caching | updates coming soon

File: 0x01-caching/100-lfu_cache.py
Comment (rezzcode) [reply 0x01-caching/100-lfu_cache.py:1]: error output ===> false :

no error was found with this code it's superb :smile: 
just overlooked onto it.

File: 0x01-caching/2-lifo_cache.py
Comment (rezzcode) [inline 0x01-caching/2-lifo_cache.py:1]: this is the file with output errors:

10/14 checks are failing which might be due to some requirement failure.

where the error lies still not known.🥲 🥲

**Review by rezzcode — COMMENTED** [review]
task two is to be checked upon

File: 0x01-caching/2-lifo_cache.py
Comment (the1Riddle) [reply 0x01-caching/2-lifo_cache.py:1]: #8

**Pull request merged**